tests/system/testdata/manifest/apply.txtar

```diff
@@ -68,6 +74,27 @@ Error: invalid YAML input: yaml: unmarshal errors:
 -- empty.yaml --
 version: v1
 
+-- wrong_integration.yaml --
+version: v1
+
+project:
+  name: my_telepathy_project
+  connections:
+    - name: my_connection
+      integration: telepathy
```

Comment:
Add a comment here that this is an unknown integration

Reply:
done